doc/source/api/solver/solution_variables.rst

```diff
@@ -0,0 +1,158 @@
+.. _ref_solution_variable_data:
+
+Solution variable data
+======================
+
+A solution variable is an array variable that holds the data for a particular
+solved field (such as pressure or velocity). You can use solution_variable_info
+and solution_variable_data objects to access Fluent solution variable info and data respectively.
+
+Accessing solution variable objects
+-----------------------------------
+
+Launch the fluent solver, and make solution variable objects available 
+(for example, by reading case and data files):
+
+.. code-block:: python
+
+  >>> import ansys.fluent.core as pyfluent
+  >>> from ansys.fluent.core import examples
+  >>> import_filename = examples.download_file("mixing_elbow.msh.h5", "pyfluent/mixing_elbow")
+  >>> solver = pyfluent.launch_fluent(mode="solver")
+  >>> solver.file.read(file_type="case", file_name=import_filename)
+
+
+The solution_variable_info and solution_variable_data objects are attributes of the solver object:
+
+.. code-block:: python
+
+  >>> solution_variable_info = solver.solution_variable_info
+  >>> solution_variable_data = solver.solution_variable_data
+
+
+Solution variable info
+----------------------
+Solution variable metadata information can be accessed via the following solution_variable_info methods:
+
+- ``get_zones_info`` for zone information.
+- ``get_solution_variables_info`` for solution variable information.
+
+Get zone information
+~~~~~~~~~~~~~~~~~~~~
+You can access zone information by calling the ``get_zones_info`` method.
+
+.. code-block:: python
+  
+  >>> zones_info = solution_variable_info.get_zones_info()
+  >>> zones_info.domains
+  ['mixture']  
+  >>>
+  >>> zones_info.zones
+  ['fluid', 'wall', 'symmetry', 'pressure-outlet-7', 'velocity-inlet-6', 'velocity-inlet-5', 'default-interior']
+  >>>
+  >>> zone_info = zones_info['wall']
+  >>>
+  >>> zone_info
+  name:wall count: 3630 zone_id:3 zone_type:wall thread_type:Face
+  >>>
+  >>> zone_info.name 
+  'wall'   
+  >>>
+  >>> zone_info.count 
+  3630 
+  >>>
+  >>> zone_info.zone_id 
+  3
+  >>>
+  >>> zone_info.zone_type 
+  'wall'  
+  
+Get solution variable information
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You can request solution variables information for a given ``domain_name`` and list of ``zone_names``
+by calling the ``get_solution_variables_info`` method.
+
+.. code-block:: python
+
+  >>> wall_fluid_info = solution_variable_info.get_solution_variables_info(zone_names=['wall' , "fluid"], domain_name="mixture")
```

Comment:
Previously `svar` would have been repeated in the object name and the method, and although it was redundant, and therefore incorrect, it would not have stuck out so much. Now that the naming has expande greatly, it really does stick, and people tend to object to this kind of thing.

One extreme re-design:
```
wall_fluid_info = solution_variable_info.get(zone_names=['wall' , "fluid"], domain_name="mixture")
```